### ISSUE TITLE
Placeholder definitions

### ISSUE THREAD

**Issue by bigbug** [gitbrent/PptxGenJS#987]
The definition of the Types does not allow settings apart from the base position props.
However, the code allows other settings like the color.
https://github.com/gitbrent/PptxGenJS/blob/273f913a8185fc979115c73f25094838bc958a3c/types/index.d.ts#L942

In the following code the "color" key is highlighted by typescript since the type definitions do not allow color. However the color in the resulting power point is shown properly. So I guess the definition just has to be adjusted properly.

```
pptx = new pptxgen();
    pptx.layout = "LAYOUT_WIDE";
    pptx.defineSlideMaster({
      title: "PLACEHOLDER_SLIDE",
      background: { color: "000000" },
      objects: [
          { text: { text: "Status Report", options: { x: 0, y: 0, w: 6, h: 0.75 } } },
          {
              placeholder: {
                  options: { name: "body", type: "body", x: 0.6, y: 1.5, w: 12, h: 5.25, color: "FFFFFF" },
                  text: "(custom placeholder text!)",
              },
          },
      ],
      slideNumber: { x: 0.3, y: "95%" },
    });
```

**Comment by travisje** [comment]
I'm running into this typing issue too and it appears in a demo that you should be able to add a `color` option  https://github.com/gitbrent/PptxGenJS/blob/master/demos/modules/masters.mjs#L127

On a sidenote, thanks for the great library @gitbrent !

**Comment by gitbrent** [comment]
Thanks @bigbug !

This issue is fixed in the upcoming v3.10.0 release:

![Screen Shot 2022-04-10 at 20 18 23](https://user-images.githubusercontent.com/7218970/162649933-9faa3a41-2518-45c0-b89c-1a77485b412d.png)
